src/main/webapp/WEB-INF/view/testdata.jsp

```diff
@@ -29,6 +29,7 @@
     <% } else{ %>
       <a href="/login">Login</a>
       <a href="/register">Register</a>
+      <a href="/users">Users</a>
```

Comment:
see comment in UsersServlet.java

There's /profile/* url already set. We can first pretend that `profile` will be renamed to `users` later. Then the url we need is already there. No need to set a new one.